newdle/calendar.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+
+from flask import g
+from icalendar import Calendar, Event, vCalAddress, vText
+from pytz import timezone
+
+
+def _email_to_vcal_address(email):
+    return vCalAddress(f'MAILTO:{email}')
+
+
+def add_participant_to_ical_event(event, participant):
```

Comment:
```suggestion
def _add_participant_to_ical_event(event, participant):
```

just like `_email_to_vcal_address` it's only used in here